src/main/java/com/softteam/vocabuilder/persistence/entity/Category.java

```diff
@@ -0,0 +1,23 @@
+package com.softteam.vocabuilder.persistence.entity;
+
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.Date;
+@Data
+@Entity
+@Table(name = "category")
```

Comment:
there are a endless discussion in table names, if singular or plural, if you have a strong feeling for singular, let's keep it like that, I personally like more plural, since it's a table where you're going to store MANY elements

Reply:
I think we can try with the plural noun.